README.md

```diff
@@ -1,6 +1,8 @@
 # AWS Support Cases Aggregator
 _AWS Support Cases aggregation for a multi-account organization_
 
+[![ci](https://github.com/Snapchat/aws-support-tickets-aggregator/ci/badge.svg?branch=master)](https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster)
```

Comment:
Apologies for the delay. I'm not very familiar with ci badges on Github.

It does appear that the proper format should include `workflows` in the url, like `https://github.com/Snapchat/aws-support-tickets-aggregator/workflows/ci/badge.svg?branch=master`. I got this URL from going to the URL linked here (`https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster`) and copying the text from "Create status badge".

Please amend the badge image URL.

Reply:
@rogergzousc thank you for looking into this. updated